spring-pulsar-spring-boot-autoconfigure/src/main/java/org/springframework/pulsar/autoconfigure/PulsarProperties.java

```diff
@@ -59,6 +60,8 @@ public class PulsarProperties {
 
 	private final Producer producer = new Producer();
 
+	private final Administration administration = new Administration();
+
 	public Map<String, Object> buildConsumerProperties() {
```

Comment:
**[TinyNit]** I think these got a little out of order before this  PR, but do you mind putting all of the `get<THING>` before the `build<THING>`s?